frontend/src/metabase/query_builder/components/QuestionActions.styled.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import { color } from "metabase/lib/colors";
+import Button from "metabase/core/components/Button";
+
+import {
+  shrinkOrExpandOnClick,
+  shrinkOrExpandDuration,
+} from "metabase/styled-components/theme/button";
+
+export const QuestionActionsContainer = styled.div`
+  border-left: 1px solid #eeecec;
```

Comment:
Please prefer to use colors from `metabase/lib/colors` not to break whitelabeling